### DISCUSSION TITLE
Define a mission for the Bankless DAO site

### DISCUSSION THREAD

**Discussion by agustinfz** [BanklessDAO/bankless-dao-website#33] [Ideas]
In order to provide focus, guidance and inspire the group of people that will contribute to the Bankless DAO website we want to define strong and concise mission which will help us define target audience/s, prioritize efforts, scope touch points wit Bankless LLC and more. We will look to web devs in the DAO and Bankless LLC stakeholders to reach consensus

The mission for the site should be a bit more specific than [that of the dao itself](https://www.bankless.community/mission) so as to provide guidance to the people developing the site. 

The mission should consider the following
- What is the goal & motivation of the Bankless DAO 
- Who is the Bankless DAO website for?
- What is the relationship between the Bankless DAO website and the Bankless LCC conten

mrfonts.eth's rough stab at a mission to get us starts:
"To become the best resource for people that want to become bankless and drive value to the Bankless DAO by growing the DeFi movement"

(Culmination of this discussion will be recorded in https://github.com/BanklessHQ/bankless-dao-website/issues/14

**Comment by wpmonty** [comment]
To support the member growth (and proselytization) of the community/dao/nation

- Make onboarding obvious, seamless, and desirable in order to help the rest of the community in claiming/acquiring their BANK
- We want to educate as many people as possible about the merits and opportunity of going bankless.

To support community initiatives & involvement

- As proposals pass, the website needs to evolve to support them - from new content & pages to entire applications
- Discord is not used by everyone; the website needs to act as a hub for members to engage with governance, new initiatives, and to see general goings-on

In short

_Mission_
**To support the growth & initiatives of the Bankless community**

_Pillars_
**Growth**
**Involvement**
**Education**

**Reply by agustinfz** [reply]
Really nice structure. Makes sense. Any particular reason to not include revenue and/or value increase for the DAO? 
Given all the passion in the DAO it is likely that it could be steered in that direction in addition to the rest of the pillars.

**Reply by wpmonty** [reply]
Value is a good pillar in that case; DAO initiatives in general should all be value drivers and the website is a medium for it - but it is a good perspective to raise Value as a specific part of the mission, even if it underlays the others.

**Reply by MrPackman4019** [reply]
Also love the break down and the pillars. One thought, instead of Involvement what about Contribution. I feel it seems to better define what the DAO is about. Thinking about what we are asking of members to do now, is to contribute to the DAO, also the DAO wants to contribute to society. Yes it is about getting involved but that involvement is for the purpose of giving your skills and your time to the movement.  What do you think?

**Reply by wpmonty** [reply]
> Also love the break down and the pillars. One thought, instead of Involvement what about Contribution.

Honestly that was probably the word I was looking for; I'm good to modify.

**Reply by freedomfflow** [reply]
Having a chronology (calendar?) of events -- probably filterable by type of events -- to help keep track of whats going on -- its like drinking from a fire hose in general to keep up.
- date and deadlines for members -- i.e. claim a badge, vote on proposals, boost a marketing campaign
- content release dates to promote hype when top notch guests will be joining podcast
... you get the idea I'm sure

We would need a login & admin UI to enter or upload content to populate this as its dynamic

**Reply by wpmonty** [reply]
> Having a chronology (calendar?) of events -- probably filterable by type of events -- to help keep track of whats going on -- its like drinking from a fire hose in general to keep up.

Definitely agree that its tough to keep up in Discord. We have the beginnings of an Events page ideation starting [here](https://github.com/BanklessHQ/bankless-dao-website/issues/32) and another one for enabling folks to [get involved](https://github.com/BanklessHQ/bankless-dao-website/issues/20).

This specific thread is primarily around a mission for the website / dev guild so we can orient ourselves as we pursue the buildout of new proposals etc. 

Great ideas there though, can't wait to see them fleshed out, proposed, and built!

**Reply by wpmonty** [reply]
Here is a revised edition, incorporating the above feedback and adjusting the perspective to that of the Developer's Guild itself:

**Mission**
To support the growth, value, and initiatives of the Bankless community and its DAO.

**Pillars**

_Drive Growth_

- To support the member growth (and proselytization) of the community and its DAO.
- To make onboarding obvious, seamless, and desirable in order to help the rest of the community in claiming/acquiring their BANK and use it in governing.

_Contribute Value_

- To support community-backed proposals and the work required to achieve them.
- To add value to the Bankless community and the BANK token

_Educate Everyone_

- To educate as many people as possible about the merits and opportunity of going bankless.
- To support translation efforts and any other effort to reach different subgroups where they commune.

**Comment by frogmonkee** [comment]
I know yall like Github, but we're having the same conversation twice. Same thing is happening here on [Discourse](https://forum.bankless.community/t/bankless-dao-constitution/). 

I don't want to tell anyone to stop using Github, it's great. But what if there was a specific channel in Discourse for devs?

**Reply by wpmonty** [reply]
No doubt room to determine the platform of discussion - jump in here to root for Discourse https://discord.com/channels/834499078434979890/834940559750725633/841031637349171220

I don't really view this as a duplicate discussion as it pertains more specifically to the dev team / website rather than the entire DAO. Naturally some overlap and happy to adjust course once the Constitution is ratified.

**Reply by frogmonkee** [reply]
Respectfully, I disagree. Questions like "Who is this website for" I definitely see as mostly you guys. But [this comment](https://github.com/BanklessHQ/bankless-dao-website/discussions/33#discussioncomment-710126) of yours is something I think forum members will want to chime in on

**Reply by agustinfz** [reply]
I think it's a matter of sequencing things right. My sense is that we should have a very broad DAO discussion as is happening on Discourse around DAO constitution. Let that settle and use that as foundational input to create a more specific website mission that is in sync w/ the overall DAO mission.

what do you all think about pausing the Website mission discussion for a few days until the Bankless constitution proposal settles out?

wrt platform I'm agnostic... whatever gets the right people involved and contributing

**Reply by MrPackman4019** [reply]
Agree we should wait.

**Reply by wpmonty** [reply]
Ok let's wait for the constitution 👍

**Comment by freedomfflow** [comment]
I think that makes a lot of sense.  No need to rush into a website mission
-- let's let things take shape for a couple of weeks and it should then
become obvious what to prioritize in terms of web development

**Comment by wpmonty** [comment]
Opening this up again in context with https://forum.bankless.community/t/mission-vision-and-values/613
